lib/ingres_sa_dialect/pyodbc.py

```diff
@@ -28,15 +27,26 @@ class Ingres_pyodbc(IngresDialect):
     def __init__(self, **kwargs):
         IngresDialect.__init__(self, **kwargs)
 
-    @classmethod
-    def dbapi(cls):
```

Comment:
Idea for next pass (i.e. no need to change for this commit). Push this logic into parent (base) class and use the pyodbc code where this is a variable NOT a string literal for lookup - to remove code duplication.